modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/IntrospectionUtils.java

```diff
@@ -369,6 +383,12 @@ public static ExpressionSupport getExpressionSupport(Expression expressionAnnota
         return expressionAnnotation != null ? expressionAnnotation.value() : SUPPORTED;
     }
 
+    public static String getAliasName(MetadataType metadataType)
+    {
+        Class<?> type = JavaTypeUtils.getType(metadataType);
+        return MuleExtensionAnnotationParser.getAliasName(type.getSimpleName(), type.getAnnotation(Alias.class));
```

Comment:
IntrospecitonUtils should not couple to MuleExtensionAnnotationParser...It should be the other way around